frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ImportCluster/ImportCluster.tsx

```diff
@@ -81,6 +81,7 @@ enum ImportMode {
   manual = 'manual',
   token = 'token',
   kubeconfig = 'kubeconfig',
+  discoveryROSA = 'discoveryROSA',
```

Comment:
I would perhaps call this something like `discoveryOCM` as it may support other cluster types eventually.